requirements.in

```diff
@@ -1,21 +1,62 @@
+#
+# This file is autogenerated by pip-compile
```

Comment:
This isn't right, this text should only be present in the autogenerated output file, not the input file